packages/lisk-p2p/src/p2p.ts

```diff
@@ -480,6 +497,52 @@ export class P2P extends EventEmitter {
 		return seedPeerUpdatedInfos;
 	}
 
+	private _pickRandomDiscoveredPeers(
+		count: number,
+	): ReadonlyArray<P2PDiscoveredPeerInfo> {
+		const discoveredPeerList: ReadonlyArray<P2PDiscoveredPeerInfo> = [
+			...this._newPeers.values(),
+			...this._triedPeers.values(),
+		]; // Peers whose values has been updated atleast once.
+
+		return selectRandomPeerSample(discoveredPeerList, count);
+	}
+
+	private _handleGetAllPeersRequest(request: P2PRequest): void {
```

Comment:
Not strictly related to this but function name should be `_handleGetPeersRequest`?